backend/file_management/file_management_helper.py

```diff
@@ -185,24 +185,32 @@ def fetch_file_contents(file_system: UnstractFileSystem, file_path: str) -> Any:
     def delete_file(file_system: UnstractFileSystem, path: str, file_name: str) -> bool:
         fs = file_system.get_fsspec_fs()
 
-        file_path = f"{path}"
-        try:
-            if file_system.path and (not path or path == "/"):
-                file_path = f"{file_system.path}/"
-        except AttributeError:
-            if fs.path and (not path or path == "/"):
-                file_path = f"{fs.path}/"
+        if hasattr(file_system, "path") and (not path or path == "/"):
+            base_path = file_system.path
+        elif hasattr(fs, "path") and (not path or path == "/"):
+            base_path = fs.path
+        else:
+            base_path = path
+        base_path = base_path if base_path.endswith("/") else base_path + "/"
 
-        file_path = file_path + "/" if not file_path.endswith("/") else file_path
+        # Directories to delete the file from
+        directories = ["", "extract/", "summarize/"]
```

Comment:
Is this function specifically intended to delete these directories? If so, could you rename the file to something more descriptive to avoid confusion?

Alternatively, it might be better to integrate this functionality directly into the prompt_studio application using a basic method from there.

Reply:
@muhammad-ali-e Not really. 
The directories has been named this way to delete the internal files in dir. The difference being the extract and summarize will have .txt files.